packages/contracts/contracts/registry/DAORegistry.sol

```diff
@@ -16,9 +16,12 @@ contract DAORegistry is InterfaceBasedRegistry {
     /// @notice The ENS subdomain registrar registering the DAO names.
```

Comment:
Do you know how much extra gas cost it takes to validate reasonable subdomain names?